pkg/client/src/app/pages/identities/components/identity-form/validateXML.ts

```diff
@@ -0,0 +1,9 @@
+const { xmllint } = require("xmllint");
+
+export const validateXML = (value, currentSchema) => {
```

Comment:
`values` and `currentSchema` need to by typed, seems they are always strings anyway.